runtime/doc/builtin.txt

```diff
@@ -9562,6 +9564,35 @@ win_id2win({expr})					*win_id2win()*
 		Can also be used as a |method|: >
 			GetWinid()->win_id2win()
 
+win_move_separator({nr}, {offset})			*win_move_separator()*
+		Move window {nr}'s vertical separator (i.e., the right border)
+		by {offset} columns, as if being dragged by the mouse. {nr}
+		can be a window number or |window-ID|. A positive {offset}
+		moves right and a negative {offset} moves left. Moving a
+		window's vertical separator will change the width of the
+		window and the width of other windows adjacent to the vertical
+		separator. The magnitude of movement may be smaller than
+		specified (e.g., as a consequence of maintaining
+		'winminwidth'). Returns TRUE if the window can be found and
```

Comment:
These lines were added as part of a recent force-push in response to a PR review [comment](https://github.com/vim/vim/pull/9486#issuecomment-1008296687).